va/va.go

```diff
@@ -341,14 +346,19 @@ func (va VAImpl) validateDNS01(task *vaTask) *core.ValidationRecord {
 func (va VAImpl) validateTLSALPN01(task *vaTask) *core.ValidationRecord {
 	portString := strconv.Itoa(va.tlsPort)
 	hostPort := net.JoinHostPort(task.Identifier, portString)
-
+	var serverNameIdentifier string
+	if task.IdentifierType == acme.IdentifierDNS {
+		serverNameIdentifier = task.Identifier
+	} else {
+		serverNameIdentifier, _ = alpnaddr(task.Identifier)
```

Comment:
This is the only call site for the `alpnaddr` function and it throws away the `error` return. I think either `alpnaddr` should be updated to not return an `error` or this call site should be changed to handle the error in some way rather than assign it to `_`.